wAbout/Report-Library-Basics.md

```diff
@@ -5,85 +5,104 @@ keywords: [report library]
-![](/images/ReportLibraryBasics/2.01UsingtheRL-Folders.png)
+![](/images/ReportLibraryBasics/ReportLibraryVersionSection.png)
 <br>
 
-**Section 2:** Select a report in the **Report Link** section. These are the links that, when clicked, will open a particular report.
+There are 4 areas within this section:
 
-![](/images/ReportLibraryBasics/2.02UsingtheRL-Report-Links.png)
+1. View Details (Brings up the Link Details window where you can view additional information about this report)
+1. View Documentation (Opens the documentation for this report. If grayed out then documentation has not been set up)
+1. Link Name & Description
+1. Versions (Displays the Version note and lists all the versions)
+
+![](/images/ReportLibraryBasics/LinkVersionSection.png)
 <br>
 
-**Section 3:** Select which version you want to use. It is recommended you use the most recent (green) version, as it is likely the best and most stable. Link Version shows the details of a specific report and any of its available versions. It is important to check Link Version before selecting **Open Link** . Each version is specified in the box at the bottom right of the Report Library window, and they are color-coded to show the stage of development (see image above).
+### Status
 
-**Note:** Depending on your user rights, all previously uploaded versions are shown:
+There are 4 statuses of a link version, each with their own distinctive color:
 
 *  _Green_ : Currently Live
+*  _Gray_ : Superseded, this is an older version of a previous live report
 *  _Yellow_ : In Testing
 *  _Red_ : In Development
-*  _Gray_ : Superseded, this is an older version of a previous live report
 
-![](/images/ReportLibraryBasics/2.03UsingtheRL-LinkVersion.png)
+![](/images/ReportLibraryBasics/Statuses.png)
 <br>
 
-**Section 4:** If you are unsure what the report is for, clicking **View Documentation** on the upper right-hand corner is particularly useful. It should include the history of version changes and a manual that explains how to use the report. If the link is gray, the documentation is not yet ready.
+## Opening a Report
 
- ![](/images/ReportLibraryBasics/2.05UsingtheRL-ViewDocumentation.png)
-<br>
+To open a report:
 
-**Section 5:** Now click **Open Link** to open the report.
+1. Select the folder
+1. Select the report
+1. Select the version
+1. Click **Open Link**
 
- ![](/images/ReportLibraryBasics/2.04UsingtheRL-OpenLink.png)
+![](/images/ReportLibraryBasics/OpenReport.png)
 <br>
 
-The selected link will open the following report.
+The report link is opened:
 
-![](/images/ReportLibraryBasics/image2017-6-16_13-50-21.png)
+![](/images/ReportLibraryBasics/ReportOpened.png)
 <br>
 
-
 ##  Navigating the Report Library
 
+Navigating the Report Library is crucial for efficient use. There are 3 main ways to navigate the Library and find the report you are looking for.
+
 ### Using the Search Feature
 
 You can search the Report Library for reports by clicking on the &lt;**Search**&gt; button:
```

Comment:
Yea this is another issue. It would be preferable in my opinion to use the <button> syntax but that would require changing a number of existing pages. I will change this so that **button** syntax is used which is consistent with most other places.